src/sdl2_compat.c

```diff
@@ -2986,8 +2987,7 @@ SDL_OpenAudio(SDL_AudioSpec *desired, SDL_AudioSpec *obtained)
     }
 
     if (g_audio_id > 0) {
-        SDL_SetError("Audio device is already opened");
-        return -1;
+        return SDL_SetError("Audio device is already opened");
```

Comment:
.. and why aren't we directly calling `SDL3_SetError` here?

Reply:
yes updated!